go.mod

```diff
@@ -2,20 +2,11 @@ module github.com/honeycombio/libhoney-go
 
 go 1.14
 
-// To keep libhoney-go working for modules and non-modules users while msgpack's
-// branch configuration doesn't meet Go modules guidelines, import paths now use
-// `gopkg.in`. This requires two module replacements:
-//   - the `github.com` line fixes the import paths in msgpack
-//   - the `gopkg.in` line fixes the import paths in libhoney-go
-// Astoundingly, `go get` actually resolves this apparent circular replacement,
-// and it survives roundtrips through `go get -u` and `go mod tidy`.
-replace (
-	github.com/vmihailenco/msgpack/v4 => gopkg.in/vmihailenco/msgpack.v4 v4.3.11
-	gopkg.in/vmihailenco/msgpack.v4 => github.com/vmihailenco/msgpack/v4 v4.3.11
-)
+replace gopkg.in/vmihailenco/msgpack.v4 => github.com/lizthegrey/msgpack v0.0.0-20200715214535-ed89c731edac
```

Comment:
Temporary, to be removed once https://github.com/vmihailenco/msgpack/pull/269 is merged and tagged as a release.